src/main/java/com/dm/dmservicesandbox/UserRequestController.java

```diff
@@ -1,14 +1,13 @@
 package com.dm.dmservicesandbox;
 
-import com.dm.dmservicesandbox.service.UserAuthenticationService;
-import org.springframework.beans.factory.annotation.Autowired;
+import com.dm.dmservicesandbox.service.UserAuthenticationServiceImpl;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("v1/user/")
```

Comment:
Thinking if we should prefer v1 at the method level, ie. /user/  at Controller level and  v1/getUserInfo at method level so that we can easily add new methods as and when, pl review and if stronger preference this way, we can discuss

Reply:
I want to remove this controller as I have added a new controller UserAuthController